readme.md

```diff
@@ -227,28 +233,67 @@ Jackson `ObjectMapper` with `GuavaModule`, `ShimJdk7Module` (same as Jackson’s
 requirement) and `Jdk8Module`. Servers must not expose parameters or return values that cannot be handled by this object
 mapper.
 
+
 #### Error propagation
-The `HttpRemotingJerseyFeature` routine installs exception mappers for `IllegalArgumentException`,
-`NoContentException`, `RuntimeException` and `WebApplicationException`. The exception mapper sets the response media
-type to `application/json` and returns as response body a JSON representation of a `SerializableError` capturing the
-message, exception name, and optionally stacktrace of the exception. Both JaxRsClient and Retrofit2Client intercept
-non-successful HTTP responses and throw a `RemoteException` wrapping the deserialized server-side `SerializableError`.
-The error name of the `RemoteException` is defined by the service API and clients should switch&dispatch based on the
-error name. The `SerializableError` format is:
+Servers should use the `ServiceException` class to propagate application-specific errors to its callers. Typically, 
```

Comment:
should link to safe-logging and mention that ServiceException implements SafeLoggable